src/jobs/database_schema_report.yml

```diff
@@ -0,0 +1,50 @@
+---
+description: >
+  Job to generate a schema report for a postgres database, using schemaspy. Runs against postgres in docker
+executor:
+  name: java
+  tag: "21.0"
+parameters:
+  host:
+    type: string
+    default: localhost
+    description: Host name of the database
+  port:
+    type: string
+    default: "5432"
+    description: Port number of the database
+  schema:
+    type: string
+    description: Schema name
+  user:
+    type: string
+    default: root
+    description: Database user name
+  password:
+    type: string
+    default: dev
+    description: Database password
+steps:
+  - run:
+      name: Install schemaspy
+      command: curl -L https://github.com/schemaspy/schemaspy/releases/download/v6.2.4/schemaspy-6.2.4.jar --output /tmp/schemaspy.jar
+  - run:
+      name: Install postgres jdbc driver
+      command: curl -L https://jdbc.postgresql.org/download/postgresql-42.7.3.jar --output /tmp/postgres-driver.jar
+  - run:
+      name: Generate database schema report
+      command: |
+        java -jar /tmp/schemaspy.jar \
+          -t pgsql \
+          -dp /tmp/postgres-driver.jar \
+          -db postgres \
```

Comment:
probably good to parameterise the database name here - the default is normally the same as the username instead

Reply:
My mistake - I thought this was the database type not the name